e2e/app/aou-elements/select-component.ts

```diff
@@ -0,0 +1,69 @@
+import {Page} from 'puppeteer';
+
+export default class SelectComponent {
+
+  private readonly page: Page;
+  private readonly name: string;
+
+  constructor(page: Page, name?: string) {
```

Comment:
Typescript constructors can be written like this:

```
constructor(
  private readonly page: Page,
  private readonly name?: string) {}
```

Then you can also remove the above type declarataions as well

Reply:
cool!